directory/directory_src.go

```diff
@@ -35,7 +36,7 @@ func (s *dirImageSource) GetManifest() ([]byte, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	return m, "", err
+	return m, manifest.GuessMIMEType(m), err
```

Comment:
I believe we need updating the doc since it makes sense for directories to try and guess the manifest mime type